proxy.go

```diff
@@ -110,10 +110,40 @@ func main() {
 		Timeout: time.Second * 10,
 	}
 
-	http.HandleFunc(settingsString[LISTEN_PATH], uploadFile)
```

Comment:
This should remain dynamic. We still only want to inject one path, as you have also defined via Cloudflare tunnel.

Reply:
In my opinion this shouldn't be dynamic as you have already configured it on the Cloudflare tunnel, so the proxy is just reciving the filtered requests with the path you want. The http server should listen to all of them, that means pointing it to "/".

Reply:
This tool is supposed to be agnostic and usable without having Cloudflare Tunnel. If we make it dynamic, people can have it as specific or broad as they want to. If we hardcode it, it can never be specific. So keeping it dynamic is better (given that already this is the case anyways, there isn't really many reasons to remove it)